en/protocol/image_transmission.md

```diff
@@ -0,0 +1,134 @@
+    MAV Component->>MAV Component: Split image into chunks.
+    MAV Component->>QGroundControl: Send image chunks.
+    QGroundControl->>QGroundControl: Receive image chunks.
+    QGroundControl->>QGroundControl: Re-assemble image and display.
+    QGroundControl->>MAV Component: Request to stop image stream
+    MAV Component->>MAV Component: Stop image preparation
+    MAV Component->>QGroundControl: Acknowledge to stop image stream
+{% endmermaid %}
+
+
+1. The communication is initiated by the *QGroundControl* with a request to start the stream. To do so, one must set the following fields in the MAVLink message:
+  * `target`: to the ID of the targeted MAV,
+  * `state`: to 0 for a request,
+  * `id`: an ID for the image stream, 
+    > **Note** For the moment, the image streamer only supports one stream per image type and therefore requires you to set the ''id'' to the same integer as the ''type'' field.
+  * `type`: any of the types in the ENUM ''MAVLINK_DATA_STREAM_TYPES'' in **mavlink.h**,
+  * `freq`: bigger than 0 for "frames per seconds", lower than 0 for "seconds per frame"
+
+It is possible to request for a specific image quality. To do so, you must set the ''quality'' field. All other fields should be zero in the initial request.
+
+1. When the targeted MAV receives the handshake request, it sends back an acknowledgment and starts the image stream at the requested framerate. The handshake ACK packet normally contains the same values as requested by the QGroundStation (`state` set to 1, because it's an ACK), and adds data about the size of the next sent image:
+  * The field ''packets'' contains the number of MAVLink `ENCAPSULATED_DATA` packets,
+  * the field ''payload'' specifies the size of the payload of each data packet (normally 252 bytes),
+  * and the ''size'' field specifies the image size in bytes.
+
+1. The image data is then split into chunks to fit into normal MAVLink messages. They are then packed into `ENCAPSULATED_DATA` packets and sent over MAVLink. Every packet contains a sequence number as well as the ID of the image stream it belongs to. The image streamer now sends periodically new images, there is no further interaction needed. Every new image comes with a new `DATA_TRANSMISSION_HANDSHAKE` ACK packet with updated image `size`, `packets` and `payload` fields. After this ACK packet, the new image arrives as a series of `ENCAPSULATED_DATA` packets.
+   > **Note** The sequence number starts at 0 for every new image of the stream.
+
+1. To stop an image stream you must send a new ''DATA_TRANSMISSION_HANDSHAKE'' request packet with the frequency set to 0. The MAV will acknowledge this by sending back an ACK packet containing the same data as in the request.
+
+### Video streaming
+
+The video transmission communication protocol is much simpler than the image streaming one: It consists of just one MAVLink message, `VIDEO_STREAM`, which is used to start and stop the video stream.
```

Comment:
I can't find this `VIDEO_STREAM` in **common.xml**. Any idea what the "right" message is?

Reply:
I thought that stuff has been replaced by:
http://mavlink.org/messages/common#MAV_CMD_VIDEO_START_STREAMING
http://mavlink.org/messages/common#MAV_CMD_VIDEO_STOP_STREAMING
http://mavlink.org/messages/common#VIDEO_STREAM_INFORMATION
http://mavlink.org/messages/common#SET_VIDEO_STREAM_SETTINGS